src/runtime_src/core/common/info_vmr.cpp

```diff
@@ -47,19 +47,16 @@ vmr_info(const xrt_core::device* device)
 {
   ptree_type pt_vmr_status_array;
   ptree_type pt_vmr_stats;
-  std::vector<std::string> vmr_status_raw;
-  try {
-    vmr_status_raw = xrt_core::device_query<xq::vmr_status>(device);
-    auto vmr_version = xrt_core::device_query<xq::extended_vmr_status>(device);
-    vmr_status_raw.insert(vmr_status_raw.begin(), vmr_version.begin(), vmr_version.end());
-  }
-  catch (const xq::exception&) {
-    // only available for versal
+  auto vmr_status = xrt_core::device_query_default<xq::vmr_status>(device);
+  auto vmr_version = xrt_core::device_query_default<xq::extended_vmr_status>(device);
```

Comment:
I am wondering what the default value is here?  It would be better to be explicit about it.